xml/System.Net.Quic/QuicConnectionOptions.xml

```diff
@@ -168,7 +170,9 @@ To use a different close error code, call <xref:System.Net.Quic.QuicConnection.C
         <ReturnType>System.TimeSpan</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The interval at which keep alive packets are sent on the connection.
+ Value <see cref="F:System.TimeSpan.Zero" /> means underlying implementation default timeout.
+ Default <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" /> means never sending keep alive packets.</summary>
```

Comment:
```suggestion</summary>
```

Reply:
Was this supposed to be a suggestion, @gewarren?

Reply:
Oh, yes I see. It was just missing an endline:
```suggestion
</summary>
```